evap/staff/tests/test_views.py

```diff
@@ -333,10 +333,25 @@ class TestUserMergeSelectionView(WebTestStaffModeWith200Check):
 
     @classmethod
     def setUpTestData(cls):
-        cls.test_users = [make_manager()]
+        cls.manager = make_manager()
+        cls.test_users = [cls.manager]
 
         baker.make(UserProfile)
 
+        cls.main_user = baker.make(UserProfile)
+        cls.other_user = baker.make(UserProfile)
```

Comment:
The previous `baker.make(UserProfile)` call was probably required to that our very basic "When you load the page it doesn't crash" check from `WebTestStaffModeWith200Check` has some data available. Now that you create additional users, I would think that this can be removed:

```suggestion
        cls.main_user = baker.make(UserProfile)
        cls.other_user = baker.make(UserProfile)
```

It might not even make sense to have the 200 check anymore, since the test case below also requires that (implicitly in the `self.app.get()`, which will fail for non-200 response codes). In that case we can also go back to inheriting from `WebTestStaffMode` here.